service/redstone-bridge/js/src/handle.ts

```
@@ -0,0 +1,43 @@
+        console.debug("Fetching", request.params);
+
+        const payloadString = await requestRedstonePayload({
+          dataServiceId: args["data-service-id"],
+          dataPackagesIds: request.params,
+          uniqueSignersCount: args["unique-signers-count"],
+          waitForAllGatewaysTimeMs: args["wait-for-all-gateways-time-ms"],
+          maxTimestampDeviationMS: args["max-timestamp-deviation-ms"],
+          authorizedSigners: getSignersForDataServiceId(
+            args["authorized-signers"] ?? args["data-service-id"],
+          ),
+        });
```

Biome (2.4.6)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)